<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>412</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnDictChooser">
       <property name="text">
        <string>Dict folders</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnModelChooser">
       <property name="text">
        <string>Choose note types</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="lwDicts">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentRow">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnParameter">
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUpdate">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAbout">
       <property name="text">
        <string>About</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnDictChooser</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>show_dictSetting_dlg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnModelChooser</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>show_models()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAbout</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>show_about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>show_dictSetting_dlg()</slot>
  <slot>show_models()</slot>
  <slot>save()</slot>
  <slot>show_about()</slot>
 </slots>
</ui>
